helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -111,6 +113,19 @@ def _read_group_stage_ids(self, stages, domain, order):
     )
     active = fields.Boolean(default=True)
 
+    def send_partner_mail(self):
+        """Send a feedback e-mail to the partner that created the ticket.
+        Note: Assigning `mail_template_id` to `helpdesk_ticket_stage_new` doesn't work
+        when the ticket is created from the portal, that's the purpose of this
+        function."""
+        created_ticket_template = self.env.ref(
```

Comment:
This has to be optional through any configuration. Maybe you can put a configuration setting for selecting the template, with this one as default.

Reply:
There is now a system variable named `helpdesk_mgmt.created_ticket_template_name`.